src/component/Calculator.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Operate from './operate';
+
+// Call the Operate function
+Operate();
+
+function Calculator() {
+  const createDigits = () => {
+    const digits = [];
+    // eslint-disable-next-line no-plusplus
+    for (let i = 1; i < 10; i++) {
+      digits.push(
+        // eslint-disable-next-line react/button-has-type
```

Comment:
- [ ] Kindly note that `operate.js` cannot be considered a component as it does not render any user interface. It contains a utility function that helps in performing calculations. As hinted previously, you can have a separate button component or create a different component containing the buttons